### PR TITLE
Add the lvcreate-option-on-storageclass proposal implementation

### PR DESCRIPTION
**Pull request by bells17**
- #627

### CODE REVIEW

**Comment by cupnes** [comment]
@bells17 
I see the following statement in [the Design Detail of the proposal](https://github.com/topolvm/topolvm/blob/207f5148aedc04d4630f5c5e9ca1478edfd9d191/docs/proposals/lvcreate-option-on-storageclass.md#design-detail), is this implemented?

> The parameter value is passed to the topolvm-controller via CreateVolume RPC.

The current PR does not include the addition of parameters to the `CreateVolumeRequest` message, which is the argument of the `CreateVolume` RPC.

**Comment by toshipp** [comment]
@cupnes 
I mean the value is embed in the parameter. He has already done it.
https://github.com/topolvm/topolvm/pull/640/files#diff-7b46685eeb2e24ed466c2d8f7c64f093cafa2b24e858407b1431c4d9e1576b47R38

**Comment by cupnes** [comment]
@toshipp Thank you. I understood.

**Review by cupnes — CHANGES_REQUESTED** [review]

**Review by llamerada-jp — CHANGES_REQUESTED** [review]

File: charts/topolvm/Chart.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@ type: application
 home: https://github.com/topolvm/topolvm
 name: topolvm
 description: Topolvm
-version: 11.0.1
+version: 11.0.2
```
[the remark this answers is not in the document]
Comment (bells17) [reply charts/topolvm/Chart.yaml]: fixed it.

File: e2e/Makefile [hunk 2]
```diff
@@ -239,14 +248,23 @@ daemonset-lvmd/create-vg:
...
+	$(SUDO) losetup -f $(BACKING_STORE)/backing_store_lvmd_8
+	$(SUDO) vgcreate -y node-myvg6 $$($(SUDO) losetup -j $(BACKING_STORE)/backing_store_lvmd_7 | cut -d: -f1) \
+		$$($(SUDO) losetup -j $(BACKING_STORE)/backing_store_lvmd_8 | cut -d: -f1)
+
 
 .PHONY: daemonset-lvmd/remove-vg
 daemonset-lvmd/remove-vg:
-	for i in $$(seq 6); do \
+	for i in $$(seq 8); do \
 		# We have only 5 VGs, but 6 PVs, since node-myvg4 uses both \
```
[the remark this answers is not in the document]
Comment (bells17) [reply e2e/Makefile]: Thank you. I deleted these comments.

**Comment by bells17** [comment]
@cupnes @llamerada-jp Thank you for your review. I fixed the points that you commented on.
Would you review this again?

**Review by llamerada-jp — APPROVED** [review]

**Review by cupnes — APPROVED** [review]

**Pull request merged**